packages/coordinator/src/screens/addAppointments/AddAppointmentsScreenContainer.tsx

```diff
@@ -13,6 +13,7 @@ export type NewSlots = {
 export default function AddAppointmentsScreenContainer() {
   const [slotsArray, setSlotsArray] = useState<NewSlots[]>([]);
   const [isSaving, setIsSaving] = useState(false);
+  const [showPopup, setShowPopup] = useState(false);
```

Comment:
I think this should be inside the component

Reply:
Moved